### PR TITLE
Qi integration

### PR DESCRIPTION
**Pull request by MichaelKostroma**
Features:
- Send Qi Asset;
- New design for the Qi sending process;
- Account switching between Qi and Quai accounts;

### CODE REVIEW

File: background/constants/coin-types.ts [hunk 1]
```diff
@@ -2,9 +2,10 @@
 // data but is important, it SHOULD NOT be imported from an external package.
 
 export const COIN_TYPES_BY_ASSET_SYMBOL = {
+  QI: 1, // FIXME
```
Comment (MichaelKostroma) [inline background/constants/coin-types.ts]: Could you kindly confirm the correct coin ID for the Qi token that we should use here?

File: ui/_locales/en/messages.json [hunk 2]
```diff
@@ -277,7 +277,7 @@
     "language": "Language",
     "bugReport": "Bug report",
     "connectedWebsites": "Connected websites",
-    "addCustomAsset": "Custom assets",
+    "addCustomAsset": "Custom QUAI assets",
```
Comment (MichaelKostroma) [inline ui/_locales/en/messages.json:280]: We've made a slight update to the title for adding a custom asset on the settings page to enhance the user experience. Does that sound good to you?

File: background/services/notifications/index.ts
Comment (ArtemBurakov) [inline background/services/notifications/index.ts:1]: Please confirm these notifications texts: header, messages. Also confirm if you want to send native Chrome notifications when the wallet is closed and a pop-up message when the wallet is open for a Qi transaction.

Comment (rileystephens28) [reply background/services/notifications/index.ts:1]: updated

File: ui/_locales/en/messages.json (hunk 2 above)
Comment (rileystephens28) [reply ui/_locales/en/messages.json:280]: This is fine

File: background/constants/coin-types.ts (hunk 1 above)
Comment (rileystephens28) [reply background/constants/coin-types.ts]: It is 969. I updated it.

**Pull request merged**